packages/zipkin/src/time.js

```diff
@@ -1,4 +1,12 @@
 // Returns the current time in epoch microseconds
-module.exports.now = function now() {
-  return new Date().getTime() * 1000;
+// if startTimestamp and startTick are present, process.hrtime is used
+// See https://nodejs.org/api/process.html#process_process_hrtime_time
+module.exports.now = function now(startTimestamp, startTick) {
+  if (startTimestamp && startTick && process && process.hrtime) {
```

Comment:
It feels that checking the existence of `process.hrtime` on every `now` call is redundant. You might do it once on module invocation and return different `now` implementations.